admin_guide/idling_applications.adoc

```diff
@@ -0,0 +1,71 @@
+[[dev-guide-idling-applications]]
+= Idling Applications
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+As an {product-title} administrator, you can idle applications in order to
+reduce the consumption of resources. 
+
+Applications are made of services, as well as resources, such as deployment
+configurations. The action of idling an application involves idling all
+accosiated resources.
+
+[[idling-applications-idling-applications]]
+== Idling Applications
+
+Idling an application involves finding the scalable resources (deployment
+configurations, replication controllers, and others) associated with a service.
+Idling an application finds the service and marks it as idled by scaling down
```

Comment:
"marks it as idled, scaling down" (remove the "by", since we don't want to give the impression that *just* scaling down a resource will mark it as idled)